website/app/styles/pages/components/radio-card.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// COMPONENTS > RADIO CARD
+
+#show-content-components-form-radio-card {
+  .doc-radio-card-list-demo {
+    margin: 0;
+    padding: 0;
+    color: #3b3d45;
+    font-size: 13px;
+    line-height: 18px;
```

Comment:
Nit: I know this is just a demo, but in general I would use unitless line-height values.

Reply:
agree – I must have copied this over from Figma, will update to `em`s

Reply:
ah, we actually don't even need these anymore, as I'm using the typography classes, will remove L13-L14